node/bin/src/prover_api/metrics.rs

```diff
@@ -1,14 +1,28 @@
 use std::time::Duration;
-use vise::{Buckets, EncodeLabelValue, Histogram, LabeledFamily, Metrics, Unit};
+use vise::{Buckets, EncodeLabelValue, Gauge, Histogram, LabeledFamily, Metrics, Unit};
 
 #[derive(Debug, Metrics)]
 #[metrics(prefix = "prover")]
 pub struct ProverMetrics {
+    #[metrics(labels = ["stage"])]
```

Comment:
Adding docstrings to all metrics would help with following up how they're used. (i.e. what's `prover_job_min_batch_number`? Minimum number it's waiting for? Minimum number it has so far?)